src/main/java/dk/kb/ginnungagap/cumulus/CumulusPreservationUtils.java

```diff
@@ -75,10 +81,12 @@ protected static void resetRepresentationMetadataGuid(CumulusRecord record) {
     protected static void initRepresentationIntellectualEntityUUID(CumulusRecord record) {
         String uuid = record.getFieldValueOrNull(Constants.FieldNames.REPRESENTATION_INTELLECTUAL_ENTITY_UUID);
         if(uuid == null || uuid.isEmpty()) {
-            record.setStringValueInField(Constants.FieldNames.REPRESENTATION_INTELLECTUAL_ENTITY_UUID, 
-                    UUID.randomUUID().toString());
+            uuid = UUID.randomUUID().toString();
+            log.trace("Setting uuid value for 'Representation intellectual guid': " + uuid);
+            record.setStringValueInField(Constants.FieldNames.REPRESENTATION_INTELLECTUAL_ENTITY_UUID,
+                    uuid);
         } else {
-            log.trace("Already has a value for the related intellectual object of the representation");
+            log.trace("Already has a value for the 'Representation intellectual guid': " + uuid);
```

Comment:
The string constant REPRESENTATION_INTELLECTUAL_ENTITY_UUID could also have been used here.